pkg/apis/configuration/v1alpha1/types.go

```diff
@@ -62,6 +62,7 @@ type TransportServer struct {
 // TransportServerSpec is the spec of the TransportServer resource.
 type TransportServerSpec struct {
 	Listener           TransportServerListener `json:"listener"`
+	ServerSnippets     string                  `json:"server-snippets"`
```

Comment:
with new resources (post-VirtualServer,  like TransortServer and Policies), we follow th camel-case convention for field names. Could we have `serverSnippets` here?